BHoM_Engine/Query/Hash.cs

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Security.Cryptography;
+using System.Reflection;
+using BH.Engine.Serialiser;
+using BH.oM.Reflection.Attributes;
+using System.ComponentModel;
+using BH.Engine.Base;
+using System.Collections;
+using System.Data;
+
+namespace BH.Engine.Base
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/ 
+
+        [Description("Computes a Hash code for the object which depends solely on its properties. The full method with additional options is in `Compute.Hash()`.")]
+        [Input("obj", "Objects the hash code should be calculated for.")]
+        public static string Hash(
+            this IObject iObj)
```

Comment:
The name of the parameters doesn't correspond to the one provided in the input attribute.

Also minor thing but wouldn't it be cleaner to have the method parameter on the same line as the rest ?

Reply:
Corrected the input description param.
I actually prefer the multiline parameters – find it much simpler to read – but if it's a matter of unblocking the PR I will change it.